regexes/device/televisions.yml

```diff
@@ -155,6 +155,8 @@ Panasonic:
       model: '$1'
     - regex: 'DIGA Webkit ([A-Z]{1}[0-9]{4})'
       model: 'DIGA $1'
+    - regex: 'SmartTV(201[89]|202[0-1])'
+      model: 'Smart TV $1'
```

Comment:
https://github.com/matomo-org/device-detector/wiki/Convention-naming-device-models#devices-containing-the-year-must-be-written-in-brackets
```yml
 model: 'Smart TV ($1)' 
 ```